core/lib/state/src/rocksdb/mod.rs

```diff
@@ -257,6 +257,12 @@ impl RocksdbStorageBuilder {
     ) -> anyhow::Result<()> {
         self.0.revert(storage, last_l1_batch_to_keep).await
     }
+
+    /// Returns the underlying storage without synchronizing it or checking that it's up-to-date.
+    /// Should not be used other than for tests.
+    pub fn build_unchecked(self) -> RocksdbStorage {
```

Comment:
Won't work; `#[cfg(test)]` is scoped by the crate, i.e., it would be visible from the `state` crate only, while we want to use it from another crate (`state_keeper`).